weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -922,7 +952,7 @@ def _select_calls_query_raw(
         for col in columns:
             if col in ["project_id", "id"]:
                 merged_cols.append(f"{col} AS {col}")
-            elif col in ["input_refs", "output_refs"]:
+            elif col in ["input_refs", "output_refs", "display_name"]:
```

Comment:
perhaps it is time to extract this list to a constant maintained near the type def?